vscode/src/test/testutils.ts

```diff
@@ -161,12 +164,12 @@ export async function waitProjectRecognized(someJavaFile: string): Promise<void>
 		const u: vscode.Uri = vscode.Uri.file(someJavaFile);
 		// clear out possible bad or negative caches.
 		return vscode.commands
-			.executeCommand(myExtension.COMMAND_PREFIX + ".clear.project.caches")
+			.executeCommand(extConstants.COMMAND_PREFIX + ".clear.project.caches")
```

Comment:
Should we replace the command names with the references from *commands.ts*?

Reply:
Done